convex/README.md

```diff
@@ -0,0 +1,90 @@
+# Welcome to your Convex functions directory!
```

Comment:
Maybe replace with instructions about how to develop on Convex in this package specifically